tests/integration/targets/aws_glue_crawler/tasks/main.yml

```diff
@@ -0,0 +1,333 @@
+---
+- name: aws_glue_crawler integration tests
+  collections:
+    - amazon.aws
+  module_defaults:
+    group/aws:
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      security_token: "{{ security_token | default(omit) }}"
+      region: "{{ aws_region }}"
+  block:
+    # AWS CLI is needed until there's a module to get info about Glue crawlers
+    - name: Install AWS CLI
+      pip:
+        name: awscli
+        state: present
+
+    - name: Create minimal Glue crawler role
+      iam_role:
+        name: "{{ glue_crawler_role_name }}"
+        assume_role_policy_document:
+          Version: "2008-10-17"
+          Statement:
+            - Action: "sts:AssumeRole"
+              Effect: Allow
+              Principal:
+                Service: glue.amazonaws.com
+        create_instance_profile: false
+        managed_policies:
+          - "arn:aws:iam::aws:policy/AWSXrayWriteOnlyAccess"
+
+    - name: Create Glue crawler (check mode)
+      aws_glue_crawler:
+        name: "{{ glue_crawler_name }}"
+        database_name: my_database
+        description: "{{ glue_crawler_description }}"
+        role: "{{ glue_crawler_role_name }}"
+        recrawl_policy:
```

Comment:
It looks like RecrawlPolicy was only added late last year. The integration tests will need to run in a virtualenv that installs a more recent boto3 than the default test image has.  (We have older defaults for now to accommodate other constraints needed to test against and with Ansible 2.9)
You can see an example of a test in a venv here: https://github.com/ansible-collections/community.aws/blob/main/tests/integration/targets/ec2_launch_template/tasks/main.yml

Reply:
@jillr Thank you, I didn't realize this was the case. I used `ec2_launch_template` tests as an example and added virtualenv logic to these tests.

Based on console output in Zuul, it looks like virtualenv gets installed correctly.

Reply:
It's now long enough ago we support this by default.